src/Networking/Network.cpp

```diff
@@ -565,6 +544,121 @@ void Network::Exit() noexcept
 #endif // HAS_NETWORKING
 }
 
+#if HAS_NETWORKING
+GCodeResult Network::ConfigureNetworkProtocol(GCodeBuffer& gb, const StringRef& reply)
+{
```

Comment:
There are several redundant calls to reprap.GetNetwork() in this function. Please remove them.